packages/docusaurus-theme-classic/src/theme/ColorModeToggle/styles.module.css

```diff
@@ -25,6 +25,10 @@
   background: var(--ifm-color-emphasis-200);
 }
 
+.navbar--dark .toggleButton:hover {
+  background: var(--ifm-color-gray-800);
+}
```

Comment:
this CSS selector does not work for me:

<img width="520" alt="CleanShot 2023-02-02 at 11 02 33@2x" src="https://user-images.githubusercontent.com/749374/216294245-80e8f7eb-e21f-40da-9856-029d961c1b98.png">


Please temporarily add a `style: 'dark'` to our site config so that we can see on the preview it works fine. We'll revert the change once it works before merging.

Reply:
hey I added `style: 'dark'` to config file

Reply:
![navbardark](https://user-images.githubusercontent.com/71703033/216318936-0324dc63-3dbf-479e-9e60-5d0b40f30d9b.png)
hey is there instead of `navbar--dark` it is `navbar--dark_Tx1g` is there any other file where I should put this style or is thre any other way to solve this

Reply:
I'm a bit new to this would really appreciate your help on this @slorber

Reply:
fixed it finally